package.json

```diff
@@ -376,6 +376,111 @@
+                  "default": 2048
+                },
+                "tokensLimit": {
+                  "type": "number",
+                  "description": "Total length of input and generated tokens, it is determined by the model.",
+                  "default": 4096
+                },
+                "maxContextTheoremsNumber": {
+                  "type": "number",
+                  "description": "Maximum number of context theorems to include in the prompt sent to the OpenAI model as examples for proof generation. Lower values reduce token usage but may decrease the likelihood of generating correct proofs.",
+                  "default": 100
+                },
+                "multiroundProfile": {
+                  "type": "object",
+                  "properties": {
+                    "maxRoundsNumber": {
+                      "type": "number",
+                      "description": "Maximum number of rounds to generate and further fix the proof. Default value is 1, which means each proof will be only generated, but not fixed.",
+                      "default": 1
+                    },
+                    "proofFixChoices": {
+                      "type": "number",
+                      "description": "Number of attempts to generate a proof fix for each proof in one round. Warning: increasing `proofFixChoices` can lead to exponential growth in generation requests if `maxRoundsNumber` is relatively large.",
+                      "default": 1
+                    },
+                    "proofFixPrompt": {
+                      "type": "string",
+                      "description": "Prompt for the proof-fix request that will be sent as a user chat message in response to an incorrect proof. It may include the `${diagnostic}` substring, which will be replaced by the actual compiler diagnostic.",
+                      "default": "Unfortunately, the last proof is not correct. Here is the compiler's feedback: `${diagnostic}`. Please, fix the proof."
+                    },
+                    "maxPreviousProofVersionsNumber": {
+                      "type": "number",
+                      "description": "Maximum number of previous proof versions to include in the proof-fix chat, each presented as a dialogue: the user's diagnostic followed by the assistant's corresponding proof attempt. The most recent proof version being fixed is always included and is not affected by this parameter.",
+                      "default": 100
+                    }
+                  },
+                  "default": {
+                    "maxRoundsNumber": 1,
+                    "proofFixChoices": 1,
+                    "proofFixPrompt": "Unfortunately, the last proof is not correct. Here is the compiler's feedback: `${diagnostic}`. Please, fix the proof.",
+                    "maxPreviousProofVersionsNumber": 100
+                  }
+                }
+              }
+            },
+            "default": [
```

Comment:
Do we want to specify the DeepSeek model by default? I kinda agree with you that it'd be nice, my only concern is just the VS Code settings being so inexplicit to set up when there are default values present (remember the feedback we got recently)... 

So I guess, yes, we might stay with the `DeepSeek` enabled by default just to provide the actual "zero setup", but maybe in the long-term we need to revise this configuration problem somehow. For example, setting no default values at all, but then immediately showing the user a warning with the 1-click button that will paste all the defaults by itself if it's needed.

Reply:
You are absolute right, I took a second thought, and, yeah, it would be more reasonable to make this an empty model set by default. Thanks!